client/src/components/shared/Info/Contracts.js

```diff
@@ -6,19 +6,19 @@ import {Badge, Button} from 'reactstrap'
 
 import ExternalLink from '../ExternalLink'
 import {ShowNumberCurrency} from '../../../services/utilities'
-import './Contracts.css'
+import './InfoButton.css'
 
 const _Contracts = ({data, toggledOn, toggle}) => {
   return (
-    <div className="contracts">
+    <div className="contracts info-button">
       <Button outline color="primary" onClick={toggle}>
         {toggledOn ? <ChevronUp aria-hidden="true" /> : <ChevronDown aria-hidden="true" />} Zmluvy{' '}
         <Badge color="primary">{data.length}</Badge>
       </Button>
       {toggledOn && (
-        <ul className="list-unstyled contracts-list">
+        <ul className="list-unstyled info-button-list">
           {data.map((contract, i) => (
-            <li key={contract.eid}>
+            <li key={i}>
```

Comment:
Right I forgot about that, thanks.
The eid is not applicable because I found cases when I entity is in 2 types of relations with a second entity (e.g. you own a company & direct a company).

Reply:
Also in the future we probably don't wont to display duplicates, but rather display 1 entity with several relation types at once. That will be some other PR.

Reply:
Sorry these are contract not relations :( ... they should have unique id on them.

Reply:
Yes, no longer applicable - there never was `eid` in contract, there are only some very long strings ...